docs/userGuide/siteConfiguration.md

```diff
@@ -36,7 +36,8 @@ Let's examine a typical `site.json` file:
   ],
   "deploy": {
     "message": "Site Update."
-  }
+  },
+  "headingIndexingLevel": 4

```

Comment:
Use the default value: 3.

Reply:
@yamgent Setting this to `3` is redundant. It's sufficient to mention the default in the table below.